src/java/org/sakaiproject/feedback/tool/entityproviders/FeedbackEntityProvider.java

```diff
@@ -1,13 +1,16 @@
 package org.sakaiproject.feedback.tool.entityproviders;
 
 import java.sql.SQLException;
+import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Date;
 import java.util.List;
 import java.util.Map;
 
 import org.apache.log4j.Logger;
 import org.apache.commons.fileupload.FileItem;
 
+import org.sakaiproject.component.cover.ServerConfigurationService;
```

Comment:
I think the Feedback tool was designed to keep all the Sakai services inside SakaiProxy to make it easier to support testing in the future.